File: problem_solving_for_beginners/string_problems/c-programming/change_case.c
#include<stdio.h>
/**
 * main - Change uppercase to lowercase and vice versa
 * Return: 0 always (success)
 */
int main(void)
{
	char str[10];
	printf("Enter a string: ");
	scanf("%s", str);

	//Write your code here
	int i;
	for (i = 0; str[i] != '\0'; i++)
	{
	    if (str[i] >= 65 && str[i] <= 90)
	        str[i] +=  32;
	    else
	        str[i] -= 32;
	}

	printf("%s\n", str);
	return 0;
}

File: problem_solving_for_beginners/string_problems/python/change_case.py
#!/usr/bin/python3
str1 = input("Enter a string: ")
strlen = 0
newstr = ""

for i in str1:
    strlen += 1

"""Change uppercase to lowercase and vice versa"""
# for i in range(strlen):
for ch in str1:
    if (ch >= 'A') and (ch <= 'Z'):
        newstr += chr(ord(ch) + 32)
    elif (ch >= 'a') and (ch <= 'z'):
        newstr += chr(ord(ch) - 32)

print(newstr)

